src/main/java/Duke.java

```diff
@@ -58,54 +58,116 @@ public static void printErrorMessage() {
                 "⠀⣼⡿⣻⣾⣿⡿⢛⣥⣾⣿⣿⣿⣿⣿⠿⣋⣴⡙⣿⣿⣿⣿⣷⣍⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠀⠀⠀⠀⠀⠘⠁⠀⣤⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡀⠀⠀⠀⠀⠰⠁⠀⠹⣿⣿⣿⣿⣿⣧⣽⣿⢸⠀");
     }
 
+    public static String[] processInputMessage(Scanner in) {
+        String input = in.nextLine();
+        String[] newInput = input.split(" ", 2);
+        return newInput;
+    }
+
+    public static String[] processEventMessage(String input) {
+        String[] eventArray = new String[3];
+        String[] inputArray = input.split(" /from ", 2);
+        if(inputArray.length == 1) {
+            eventArray[0] = "";
+            return eventArray;
+        }
+        eventArray[0] = inputArray[0];
+        inputArray = inputArray[1].split(" /to ", 2);
+        if(inputArray.length == 1) {
+            eventArray[0] = "";
+            return eventArray;
+        }
+        eventArray[1] = inputArray[0];
+        eventArray[2] = inputArray[1];
+        return eventArray;
+    }
+
+    public static String[] processDeadlineMessage(String input) {
+        String[] deadlineArray = input.split(" /by ", 2);
+        if(deadlineArray .length == 1) {
```

Comment:
There is an extra space before .length, maybe you can try using the ctrl + alt + L command to auto reformat